File: app/src/main/java/com/BlizzardArmory/warcraft/Talents.java
package com.BlizzardArmory.warcraft;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import org.apache.commons.lang.builder.ToStringBuilder;

public class Talents {

    @SerializedName("tier")
    @Expose
    private Integer tier;
    @SerializedName("column")
    @Expose
    private Integer column;
    @SerializedName("spell")
    @Expose
    private Spell spell;
    @SerializedName("spec")
    @Expose
    private Spec spec;

    public Integer getTier() {
        return tier;
    }

    public void setTier(Integer tier) {
        this.tier = tier;
    }

    public Integer getColumn() {
        return column;
    }

    public void setColumn(Integer column) {
        this.column = column;
    }

    public Spell getSpell() {
        return spell;
    }

    public void setSpell(Spell spell) {
        this.spell = spell;
    }

    public Spec getSpec() {
        return spec;
    }

    public void setSpec(Spec spec) {
        this.spec = spec;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("tier", tier).append("column", column).append("spell", spell).append("spec", spec).toString();
    }
}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/talents/playerspec/Talents.kt
package com.BlizzardArmory.model.warcraft.talents.playerspec

import com.BlizzardArmory.model.warcraft.common.SpellTooltip
import com.google.gson.annotations.SerializedName


data class Talents(
    @SerializedName("talent") var talent: Talent,
    @SerializedName("spell_tooltip") var spellTooltip: SpellTooltip,
    @SerializedName("talent_rank") var talentRank: Int
)